src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Arrivals/IncidentStepDef.scala

```diff
@@ -86,14 +86,14 @@ class IncidentStepDef extends BaseStepDef {
       .submitPage()
   }
 
-  And("""^(?:I )?click the (.*) link on the You have added (.*) incidents? page""") {
+  And("""^(?:I )?click the (.*) link on the You have added (.*) incidents? page$""") {
     (sectionLink: String, numberOfIncidentsItems: String) =>
       AddAnotherIncidentPage
         .loadPage(numberOfIncidentsItems)
         .clickByPartialLinkText(sectionLink)
   }
 
-  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove incident (.*) page""") {
+  And("""^(?:I )?click radio option (.*) on the Are you sure you want to remove incident (.*) page$""") {
```

Comment:
Keep the consistency?
`"""^(?:I )?click radio option (.*) on the 'Are you sure you want to remove incident (.*)\?' page$"""`

Adding '' and \?